### ISSUE TITLE
babel-node broken by 3.0.0-beta.3

### ISSUE THREAD

**Issue by pwhipp** [gaearon/react-hot-loader#359]
The entry in the .bablrc plugins: "react-hot-loader/babel" is needed for the hot loading but it breaks the babel-node repl -  everything in the repl just returns undefined. Removing the plugin  restores the functionality making this a 100% reproducible issue for me.

**Comment by iam4x** [comment]
Was hard to find the culprit, but @pwhipp is right: 

```
ERROR in ./modules/inputs/order-stock-items/index.jsx
Module parse failed: /Users/iam4x/save/ivi-front/node_modules/babel-loader/index.js?{"presets":["react",["es2015",{"modules":false}],"stage-0"],"plugins":["import-glob","transform-async-to-generator","transform-decorators-legacy","transform-runtime","lodash","react-hot-loader/babel","typecheck"]}!/Users/iam4x/save/ivi-front/modules/inputs/order-stock-items/index.jsx The keyword 'await' is reserved (121:8)
You may need an appropriate loader to handle this file type.
|         var url = STOCKS_API + '/stock_items/' + this.pendingItemTrackingCode;
|         var data = { location_type: 'Order', location_id: id };
|         await axios({ method: 'put', url: url, data: data });
|         await this.refreshOrder();
|       } catch (error) {
 @ ./modules/inputs/index.jsx 40:0-50
 @ ./modules/filters/index.jsx
 @ ./modules/ivi-resource/list/index.jsx
 @ ./modules/ivi-resource/index.jsx
 @ ./app/savers/savers.routes.jsx
 @ ./core/router/index.jsx
 @ ./core/index.jsx
 @ multi app
```

**Comment by calesce** [comment]
Thanks for catching! I wonder if we can opt out of the plugin if we detect we're in the CLI.

**Comment by calesce** [comment]
Fixed by #374 and out in `3.0.0-beta.5`.
